modules/scripting/src/main/java/org/mule/module/scripting/builders/ScriptConfigurationBuilder.java

```diff
@@ -66,6 +67,9 @@ public ScriptConfigurationBuilder(String scriptEngineName, String[] configResour
     protected void doConfigure(MuleContext muleContext) throws Exception
     {
         this.muleContext = muleContext;
+        SimpleRegistry registry = new SimpleRegistry(muleContext);
```

Comment:
Why is this code adding a registry that is supposed to not be used on production?